core-web/libs/portlets/dot-locales/portlet/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+    "compilerOptions": {
+        "target": "es2022",
+        "useDefineForClassFields": false,
+        "forceConsistentCasingInFileNames": true,
+        "strict": true,
+        "noImplicitOverride": true,
+        "noPropertyAccessFromIndexSignature": true,
+        "noImplicitReturns": true,
+        "noFallthroughCasesInSwitch": true
+    },
```

Comment:
Doesn't this overwrite the `compilerOptions` we have globally in the `tsconfig.base.json`?

Reply:
no, those properties are not present.